docs/intelligentapps/evaluation.md

```diff
@@ -0,0 +1,36 @@
+---
+Order: 5
+Area: intelligentapps
+TOCTitle: Evaluation
+ContentId:
+PageTitle: AI Evaluation
+DateApproved:
+MetaDescription: Import a dataset with LLMs or SLMs output or rerun it for the queries. Run evaluation job for the popular evaluators like F1 score, relevance, coherence, similarity... find, visualize, and compare the evaluation results in tables or charts.
+MetaSocialImage:
+---
+
+# Evaluation Overview
+
+AI engineers often need to evaluate models with different parameters, different prompts in a dataset for comparing to ground truth and generate values from the comparisons. AI Toolkit allows you do perform evaluations with minimal efforts.
+
+## Prepare an evaluation job
```

Comment:
What does it mean to prepare an evaluation job? Is this just a technical step before you can run, or is something actually being done? If nothing special, include it as part of starting an evaluation job.
At least describe what this is.